infrastructure/provider.tf

```diff
@@ -25,7 +25,8 @@ provider "azurerm" {
 
 provider "azurerm" {
   alias           = "tooling"
-  subscription_id = var.tooling_config.subscription_id
+  subscription_id = var.front_door_config.subscription_id
+
```

Comment:
Let's not change this - it will break other instances of tooling and we shouldn't need to do it. See other comments.